test/secp256k1.webcrypto.test.js

```diff
@@ -0,0 +1,555 @@
+import { webcrypto } from 'node:crypto';
+// @ts-ignore
+globalThis.crypto = webcrypto;
```

Comment:
What if we make the file import `secp256k1.test.js` after those 3 lines? Do you think it would work?

Reply:
> Just doing an import doesn't execute any tests

what if you run `should.run()`?

```
// webcrypto shim
// import test
should.run()
```